--- /usr/share/lipstick-jolla-home-qt5/eventsview/EventsViewSidebar.qml	2017-03-16 14:11:25.226141238 +0100
+++ /usr/share/lipstick-jolla-home-qt5/eventsview/EventsViewSidebar.qml	2017-03-16 14:13:17.789955645 +0100
@@ -60,9 +60,9 @@
             top: parent.top
             topMargin: 2*Theme.paddingLarge
             left: mainSeparator.right
-            leftMargin: Theme.paddingLarge
+            leftMargin: Theme.paddingSmall
             right: parent.right
-            rightMargin: Theme.paddingLarge
+            rightMargin: Theme.paddingSmall
         }
 
         SimSelector {
@@ -81,9 +81,10 @@
             id: favoriteSettingsLoader
 
             width: parent.width
-            height: item.height + 3*Theme.paddingLarge
+            height: item.height + Theme.paddingLarge
             active: shortcutsEnabled.value
             pager: eventViewList
+            showListFavorites: true
 
             ConfigurationValue {
                 id: shortcutsEnabled
@@ -111,6 +112,7 @@
         }
 
         IconButton {
+            visible: false
             anchors.right: parent.right
             icon.source: "image://theme/icon-s-setting"
  
--- /usr/share/lipstick-jolla-home-qt5/main/Desktop.qml	2017-03-09 02:18:02.697963356 +0100
+++ /usr/share/lipstick-jolla-home-qt5/main/Desktop.qml	2017-03-09 02:18:25.081678202 +0100
@@ -25,7 +25,7 @@
     property bool eventsViewVisible
     property bool eventsSubviewVisible
     property bool eventsViewNeedsConnectivity
-    property bool showEventsViewSidebar: Screen.sizeCategory >= Screen.Large
+    property bool showEventsViewSidebar: Screen.sizeCategory >= Screen.Small
     property real eventsViewSidebarWidth
     property real eventsColumnMaxWidth: showEventsViewSidebar ? Math.max(Screen.width, Screen.height) / 3 * 2
                                                               : Math.min(Screen.width, Screen.height)

--- /usr/share/lipstick-jolla-home-qt5/eventsview/FavoriteSettingsLoader.qml	2017-03-16 14:22:42.916115747 +0100
+++ /usr/share/lipstick-jolla-home-qt5/eventsview/FavoriteSettingsLoader.qml	2017-03-16 14:23:27.930187990 +0100
@@ -11,12 +11,20 @@
 import org.nemomobile.dbus 2.0
 import org.nemomobile.lipstick 0.1
 import "../main"
+import org.nemomobile.configuration 1.0
 
 Loader {
     id: root
 
+    ConfigurationGroup {
+        id: toggles
+        path: "/desktop/lipstick/sailfishos-patch-tabletui"
+        property real portrait: 3
+        property real landscape: 3
+    }
+
     property real firstColumnVCenterOffset: width / columns / 2
-    property int columns: 3
+    property int columns: isLandscape ? toggles.landscape : toggles.portrait
     property bool showListFavorites
     property Item pager
     property int padding

--- /usr/lib/qt5/qml/Sailfish/Silica/Slider.qml	2017-03-18 16:43:40.059274969 +0100
+++ /usr/lib/qt5/qml/Sailfish/Silica/Slider.qml	2017-03-18 16:43:17.825583726 +0100
@@ -49,8 +49,8 @@
     property alias label: labelText.text
     property bool down: pressed && !DragFilter.canceled && !_cancel
     property bool highlighted: down
-    property real leftMargin: Math.round(Screen.width/8)
-    property real rightMargin: Math.round(Screen.width/8)
+    property real leftMargin: Math.round(parent.width/8)
+    property real rightMargin: Math.round(parent.width/8)
     property color highlightColor: Theme.highlightColor
     property color secondaryHighlightColor: Theme.secondaryHighlightColor
