--- /usr/share/lipstick-jolla-home-qt5/switcher/Switcher.qml	2017-03-09 02:27:11.680311016 +0100
+++ /usr/share/lipstick-jolla-home-qt5/switcher/Switcher.qml	2017-03-09 02:27:50.434261886 +0100
@@ -350,7 +350,7 @@
             readonly property real minimumHeight: switcherRoot.height - statusBarHeight - ((largeScreen ? 5 : 3) * Theme.paddingLarge)
             readonly property real maximumWidth: switcherRoot.width - 2*Theme.paddingLarge
 
-            readonly property bool allowSmallCovers: !largeScreen
+            readonly property bool allowSmallCovers: false
             readonly property int smallColumns: (maximumWidth + spacing) / (Theme.coverSizeSmall.width + spacing)
 
             readonly property int largeColumns: Math.min(4, (maximumWidth + spacing) / (Theme.coverSizeLarge.width + spacing))
